msteams-platform/m365-apps/using-teams-client-sdk-preview.md

```diff
@@ -0,0 +1,365 @@
+
+## Updating to the Teams client SDK v2 Preview
+
+The easiest way to update your Teams app to use the TeamsJS SDK v2 Preview is to use the [Teams Toolkit extension](https://aka.ms/teams-toolkit) for Visual Studio Code. This section will walk you through the steps to do that. If you prefer to manually update your code, see the [Callbacks converted to promises](#callbacks-converted-to-promises) and [APIs organized into capabilities](#apis-organized-into-capabilities) sections for more details on required API changes.
+
+### 1. Install the latest Teams Toolkit VS Code extension
+
+In the *Visual Studio Code Extensions Marketplace*, search for **Teams Toolkit** and install version `2.10.0` or later. The toolkit provides two commands to assist the process:
+
+1. A command to update your manifest schema
+1. A command to update your SDK references and call sites
+
+These are the two key updates you'll need to run a Teams personal tab app in other Microsoft 365 applications.
+
+### 2. Updating the manifest
+
+# [Teams Toolkit](#tab/manifest-teams-toolkit)
+
+1. Open the *Command palette*: `Ctrl+Shift+P`
+1. Run the `Teams: Upgrade Teams manifest to support Outlook and Office apps` command and select your app manifest file. Changes will be made in place.
+
+# [Manual steps](#tab/manifest-manual)
+
+Open your Teams app manifest and update the `$schema` and `manifestVersion` with the following values:
+
+```json
+{
+    "$schema" : "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+    "manifestVersion" : "m365DevPreview"
+}
+```
+---
+
+If you used Teams Toolkit to create your personal app, you can also use it to validate the changes to your manifest file and identify any errors. Open the command palette `Ctrl+Shift+P` and find **Teams: Validate manifest file** or select the option from the Deployment menu of the Teams Toolkit (look for the Teams icon on the left side of Visual Studio Code).
+
+:::image type="content" source="images/toolkit-validate-manifest-file.png" alt-text="Teams Toolkit 'Validate manifest file' option under 'Deployment' menu":::
+
+### 2. Update SDK references
+
+To run in Outlook and Office, your app will need to depend on the [npm package](https://www.npmjs.com/package/@microsoft/teams-js/v/2.0.0-beta.0) `@microsoft/teams-js@2.0.0-beta.0` or higher. To perform these steps manually, and for more information on the API changes, see the following sections on [Callbacks converted to promises](#callbacks-converted-to-promises) and [APIs organized into capabilities](#apis-organized-into-capabilities).
```

Comment:
We should either update the version to be `@microsoft/teams-js@2.0.0-beta.1` or use `@microsoft/teams-js@next` since  we have deprecated 2.0.0-beta.0.